_opensearch/install/quickstart.md

```diff
@@ -0,0 +1,162 @@
+
+1. Before running OpenSearch on your machine, you should review and apply these [important system settings]({{site.url}}{{site.baseurl}}/opensearch/install/important-settings/).
+    - Disable memory paging and swapping performance on the host to improve performance.
+        ```bash
+        sudo swapoff -a
+        ```
+    - Increase the number of memory maps available to OpenSearch.
+        ```bash
+        # Edit the sysctl config file.
+        sudo vi /etc/sysctl.conf
+
+        # Define the max map count.
+        vm.max_map_count=262144
+
+        # Reload the kernel parameters.
+        sudo sysctl -p
+        ```  
+1. Download the sample Compose file to your host. You can download the file with command line utilities like `curl` and `wget`, or you can manually copy [docker-compose.yml](https://github.com/opensearch-project/documentation-website/tree/{{site.opensearch_version}}/assets/examples/docker-compose.yml) from the OpenSearch Project documentation-website repository using a web browser.
+    ```bash
+    # Using cURL:
+    curl -O https://github.com/opensearch-project/documentation-website/tree/{{site.opensearch_version}}/assets/examples/docker-compose.yml
+
+    # Using wget:
+    wget https://github.com/opensearch-project/documentation-website/tree/{{site.opensearch_version}}/assets/examples/docker-compose.yml
+    ```
+1. In your terminal application, navigate to the directory containing the `docker-compose.yml` file you just downloaded, and run the following command to create and start the cluster as a background process.
+    ```bash
+    docker-compose up -d
+    ```
+1. Confirm that the containers are running.
+    ```bash
+    docker-compose ps
+    ```
+1. Query the OpenSearch REST API to verify that the service is running. You should use `-k` (also written as `--insecure`) to disable host name checking because the default security configuration uses demo certificates. Use `-u` to pass the default username and password (`admin:admin`).
+    ```bash
+    curl https://localhost:9200 -ku admin:admin
+    ```
+    - Sample response:
+    ```json
+    {
+    "name" : "opensearch-node1",
+    "cluster_name" : "opensearch-cluster",
+    "cluster_uuid" : "Cd7SL5ysRSyuau325M3h9w",
+    "version" : {
+        "distribution" : "opensearch",
+        "number" : "2.3.0",
+        "build_type" : "tar",
+        "build_hash" : "6f6e84ebc54af31a976f53af36a5c69d474a5140",
+        "build_date" : "2022-09-09T00:07:12.137133581Z",
+        "build_snapshot" : false,
+        "lucene_version" : "9.3.0",
+        "minimum_wire_compatibility_version" : "7.10.0",
+        "minimum_index_compatibility_version" : "7.0.0"
+    },
+    "tagline" : "The OpenSearch Project: https://opensearch.org/"
+    }
+    ```
```

Comment:
Since we are using sample docker-compose.yml here, the Dashboards would be up and running as well. In the next step, describe accessing Dashboards by going to localhost:5601 on your browser and login with admin:admin credentials

Reply:
ignore if you think this is not needed as Dashboards is already covered elsewhere